### ISSUE TITLE
Web support?

### ISSUE THREAD

**Issue by likeSo** [alexbrazier/react-native-network-logger#52]
**Feature Request**

Web browsers have a network logger. The only thing that this library should do is "Nothing"!
When i use this library with `react-native-web`,i got this error:
```sh
Failed to compile.
/Users/xxx/xxx/xxx/node_modules/react-native/Libraries/Network/XMLHttpRequest.js
Module not found: Can't resolve './RCTNetworking' in '/Users/xxx/xxx/SellerCenterApp/node_modules/react-native/Libraries/Network'
```
I finally solve this error by **DELETE all references of this library** when i am building for the web.

**Additional context**

On the web platform, this library should do nothing, because the browsers have their logging already.

**Comment by alexbrazier** [comment]
Hi @likeSo, thanks for raising the issue. Do you have any ideas on a fix for this or an example repo to recreate the issue for me to test and fix?

**Comment by likeSo** [comment]
@alexbrazier I was trying to reproduce the problem on the expo's https://snack.expo.dev/ , but I can't even add this library as a dependency. It's the same error I had
![image](https://user-images.githubusercontent.com/12247340/127472960-8b334ff5-73b0-47da-959e-471806cbf9a4.png)

**Comment by likeSo** [comment]
I think the problem might be that you're using one React Native's inner library:
`import XHRInterceptor from 'react-native/Libraries/Network/XHRInterceptor';`

The Internal API  does not support the web platform. Maybe you should have a try on dynamic import?

**Comment by alexbrazier** [comment]
@likeSo thanks for the details. I think I've managed to fix this in [1.10.1](https://github.com/alexbrazier/react-native-network-logger/releases/tag/v1.10.1) (at least on expo snack). Are you able to test it with your project to see if it is working now? Feel free to reopen if there are any issues!

**Comment by likeSo** [comment]
@alexbrazier Yes i can test with it when i am free
